### PR TITLE
UNCID-943 weekly count fix, Country Acronym fix

### PR DESCRIPTION
**Pull request by uiftikhar**

### CODE REVIEW

File: src/js/controllers/NumbersCtrl.js [hunk 1]
```diff
@@ -325,6 +322,8 @@ function NumbersCtrl($scope) {
         unique_Org = org_names.filter(onlyUnique);
         $scope.sortedCountries = unique;
         $scope.sortedOrg = unique_Org;
+        var ONE_WEEK = 1000 * 60 * 60 * 24 * 7;
```
Comment (guillaumev) [inline src/js/controllers/NumbersCtrl.js]: @uiftikhar Is this a variable or a constant ?

File: src/js/controllers/NumbersCtrl.js [hunk 2]
```diff
@@ -570,31 +575,40 @@ function NumbersCtrl($scope) {
...
-                $scope.sixMonthlyDetails[i].country = 'CAR';
-            if($scope.sixMonthlyDetails[i].country.toString().includes(country3))
-                $scope.sixMonthlyDetails[i].coucountryntryName = 'Palestine';            
+            {
+                if($scope.sixMonthlyDetails[i].country.toString().includes(country1))
+                   $scope.sixMonthlyDetails[i].country = 'DRC';
+                if($scope.sixMonthlyDetails[i].country.toString().includes(country2))
+                   $scope.sixMonthlyDetails[i].country = 'CAR';
+                if($scope.sixMonthlyDetails[i].country.toString().includes(country3))
+                   $scope.sixMonthlyDetails[i].coucountryntryName = 'Palestine';
```
Comment (guillaumev) [inline src/js/controllers/NumbersCtrl.js]: @uiftikhar Are you sure you have the right variable name on this line ?

File: src/js/controllers/NumbersCtrl.js (hunk 1 above)
Comment (uiftikhar) [inline src/js/controllers/NumbersCtrl.js]: Actually this should have been deleted. It's not being used anywhere. Ill fix this

File: src/js/controllers/NumbersCtrl.js (hunk 2 above)
Comment (uiftikhar) [inline src/js/controllers/NumbersCtrl.js]: Fixed now. Do not know how that happened

**Pull request merged**